shared/macros.jinja

```diff
@@ -598,7 +618,7 @@ ocil_clause: "the correct value is not returned"
 
 
 {{% macro body_of_warning_about_dependent_rule(rule_id, why) -%}}
-        When selecting this rule in a profile, 
+        When selecting this rule in a profile,
```

Comment:
This change doesn't seem to make a difference, this is what is in the DS:
```
                <ns10:warning category="dependency" xml:lang="en-US">When selecting this rule in a profile,
    make sure that rule with ID <html:code/> is selected as well: because we need it</ns10:warning>
```
and rendered HTML guide:
![warning_body_macro](https://user-images.githubusercontent.com/7460169/66481311-e938ee00-eaa0-11e9-9aec-1b9a8fe7a629.png)
Note that the rule ID is not correctly shown, either I tested the macro incorrectly, or there is bug. It doesn't affect the line continuation, though.

@matejak I'd rather keep the PR changes focused and not touch this line.

Reply:
OK, I have just removed a trailing whitespace. I have reverted that by a rebase.

Reply:
Thanks, sorry If I was a bit pedantic here.